frida-gum/src/stalker/transformer.rs

```diff
@@ -59,7 +60,7 @@ use frida_gum_sys::cs_insn;
 
 pub struct Instruction<'a> {
     parent: *mut frida_gum_sys::GumStalkerIterator,
-    instr: *const cs_insn,
+    instr: Insn<'a>,
```

Comment:
Hmm, should we own this instruction? What happens when it `drop`s? Does it free the instruction?

Reply:
SFAIK `Insn` doesn't drop the `cs_insn`, as it is generally owned by the `Insstructions` iterator.